ibm_catalog.json

```diff
@@ -0,0 +1,114 @@
+      ],
+      "short_description": "Implements an instance of IBM Cloud Databases for Redis.",
+      "long_description": "Use this module to provision and configure an instance of IBM Cloud Databases for Redis.\n\n#### Usage\n```hcl\nprovider \"ibm\" {\n  ibmcloud_api_key = \"XXXXXXXXXX\"\n  region           = \"us-south\"\n}\n\nmodule \"redis\" {\n  # Replace “X.X.X” with a release version to lock into a specific release\n  source            = \"https://cm.globalcatalog.cloud.ibm.com/api/v1-beta/offering/source?archive=tgz&kind=terraform&name=terraform-ibm-icd-redis&version=X.X.X\"\n  resource_group_id = \"xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX\"\n  region            = \"us-south\"\n  instance_name     = \"my-redis-instance\"\n}\n```",
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-icd-redis/blob/main/README.md",
+      "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-icd-redis/main/images/redis_icon.svg",
+      "features": [
+        {
+          "title": "Create instance of Databases for Redis",
+          "description": "Creates an instance of IBM Cloud Databases for Redis."
+        },
+        {
+          "title": "Managing users and roles",
+          "description": "Manage IAM service credentials as well as Databases for Redis admin and default user credentials and roles."
+        },
+        {
+          "title": "Autoscaling instance",
+          "description": "Supports rules that allow the Cloud Databases instance to increase resources in response to usage."
+        },
+        {
+          "title": "Create service credentials roles",
+          "description": "Create service credentials roles for the instance of Databases for Redis."
+        },
+        {
+          "title": "Configure Databases for Redis instance",
+          "description": "Configure Databases for Redis settings to tune database based on the use-case."
+        },
+        {
+          "title": "Scaling disk, memory, and CPU",
+          "description": "Adjust the Cloud Databases resources to suit workloads and other needs."
+        },
+        {
+          "title": "KMS encryption",
+          "description": "Supports key management encryption (BYOK and KYOK), including service-to-service authorization policies with KMS."
+        },
+        {
+          "title": "CBR",
+          "description": "Create context-based restriction (CBR) rules for both Databases for Redis instance."
+        }
+      ],
+      "flavors": [
+        {
+          "label": "Basic",
+          "name": "basic",
+          "working_directory": "examples/basic",
+          "architecture": {
+            "diagrams": [
+              {
+                "diagram": {
+                  "caption": " ",
+                  "url": "https://upload.wikimedia.org/wikipedia/commons/1/1d/No_image.svg",
+                  "type": "image/svg+xml"
+                },
+                "description": " "
```

Comment:
The description here should match the description at the top of the example readme